dotnet-client-libraries/BenchmarkDotNetWithUnitTests/Tests/StringConcatBenchmarkLiveTest.cs

```diff
@@ -0,0 +1,52 @@
+using System.Collections.Immutable;
+using BenchmarkDotNet.Reports;
+
+namespace Tests;
+
+public class StringConcatBenchmarkLiveTest : IClassFixture<BenchmarkFixture>
+{
+    private readonly ImmutableArray<BenchmarkReport> _benchmarkReports;
+    private readonly BenchmarkReport _stringInterpolationReport;
+
+    public StringConcatBenchmarkLiveTest(BenchmarkFixture benchmarkFixture)
+    {
+        _benchmarkReports = benchmarkFixture.BenchmarkSummary.Reports;
+        _stringInterpolationReport = _benchmarkReports.First(x =>
+            x.BenchmarkCase.Descriptor.DisplayInfo == "StringConcatBenchmarks.StringInterpolation");
+    }
+
+    [Fact]
+    public void WhenBenchmarkTestsAreRun_ThenFourCasesMustBeExecuted()
+    {
+        var benchmarkCases = _benchmarkReports.Length;
+        Assert.Equal(4, benchmarkCases);
+    }
+
+    [Fact]
+    public void WhenStringInterpolationCaseIsExecuted_ThenItShouldNotTakeMoreThanFifteenNanoSecs()
+    {
+        var stats = _stringInterpolationReport.ResultStatistics;
+        Assert.True(stats is { Mean: < 15 }, $"Mean was {stats.Mean}");
+    }
+
+    [Fact]
+    public void WhenStringInterpolationCaseIsExecuted_ThenItShouldNotConsumeMemoryMoreThanMaxAllocation()
+    {
+        const int maxAllocation = 1342178216;
+        var memoryStats = _stringInterpolationReport.GcStats;
+        var stringInterpolationCase = _stringInterpolationReport.BenchmarkCase;
+        var allocation = memoryStats.GetBytesAllocatedPerOperation(stringInterpolationCase);
+        Assert.True(allocation <= maxAllocation, $"Allocation was {allocation}");
+        var totalAllocatedBytes = memoryStats.GetTotalAllocatedBytes(true);
+        Assert.True(totalAllocatedBytes <= maxAllocation,
```

Comment:
Add blank line

Reply:
Already done in previous commit
https://github.com/goldytech/CodeMazeGuides/blob/benchmarkdotnet-with-unit-tests/dotnet-client-libraries/BenchmarkDotNetWithUnitTests/Tests/StringConcatBenchmarkLiveTest.cs